lib/GenomeFileUtil/core/GenomeToGenbank.py

```diff
@@ -162,10 +162,8 @@ def __init__(self, genome=None):
         for contig_tuple in contigs_tuples:
             cid = contig_tuple[0]
             feature_ids_by_contig[cid] = {}
-            print("CONTIG ID : " + str(cid))
```

Comment:
I agree. I will comment them out. They do make useful debugging comments if we have other issues.